server/src/domain/person/person.service.ts

```diff
@@ -199,6 +199,11 @@ export class PersonService {
 
     const { name, birthDate, isHidden, featureFaceAssetId: assetId } = dto;
 
+    // Check if the birthDate is in the future
+    if (birthDate && new Date(birthDate) > new Date()) {
+        throw new BadRequestException('Date of birth cannot be in the future');
+    }
```

Comment:
Can you add a unit test for this ? Should be in `person.service.spec.ts`

Reply:
Thanks @martabal. The tests are added now.